numba/cpython/builtins.py

```diff
@@ -550,8 +550,29 @@ def indval_min(indval1, indval2):
     if isinstance(indval1, IndexValueType) and \
        isinstance(indval2, IndexValueType):
         def min_impl(indval1, indval2):
-            if indval1.value > indval2.value:
+            # nan check
```

Comment:
Thanks for fixing this, suggest adding unit tests that include `np.nan` values as part of their input to `np.argmax`/`np.argmin`.

Reply:
Done.